docs/getting_started.md

```diff
@@ -1,9 +1,33 @@
 Getting Started
 ================
-1. Run the `setup.sh` script to find & replace placeholders with information about your project
-2. Fill out `README.md` with details about your project
-3. Edit documentation configuration in `conf.py`
-    - For the full list of available options, see [the official documentation][Sphinx-docs].
-4. Rewrite this page with instructions for how to get started with your project
+
+### Setup
+1. Ensure batteries are charged
+2. Connect flight computers via USB cable
+3. Start AltOS
+4. Download and delete all on-board flight logs
+5. Configure conditions for recovery and staging events
+6. Calibrate accelerometers
+--For RF enabled flight computer only--
+7. Disconnect flight computer and connect Teledongle
+8. Connect battery to flight computer and turn on
+9. Configure channel and click 'monitor idle'
+10. Ensure flight computer telemetry is operational
+
+### Arming
+1. Ensure all avionics are off before moving rocket from integration are to launch pad
+2. Slide rocket into rails and set vertical
```

Comment:
Might want to turn Telemega on in Idle mode (rocket has to be horizontal) so that it can get reset into Pad mode from a safe distance.

Reply:
Telemega has to be turned on while vertical to arm itself, hence telemega off -> on pad -> telemega on for arming